plugin.video.rocketbeans/resources/lib/plugin.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+import routing
+
+from resources.data import config
+from resources.lib.youtube import YoutubeStream
+from resources.lib.twitch import TwitchStream
+from xbmcgui import ListItem
+from xbmcplugin import addDirectoryItem, endOfDirectory, setContent
+from xbmcaddon import Addon
+
+
+plugin = routing.Plugin()
+setContent(plugin.handle, 'videos')
+
+livestream_thumbnail = None
+livestream_url = None
+
+__addon__ = Addon()
+
+def getString(string_id):
+    return __addon__.getLocalizedString(string_id).encode('utf-8', 'ignore')
```

Comment:
You don't need to do encode in Python 3 and certainly should use `ignore` error handler with great care.

Reply:
Yeah, I don't think I needed to touch this part. Seems to still work.